source/developers/sts-for-operator.rst

```diff
@@ -0,0 +1,186 @@
+4. The MinIO Operator checks for :ref:`Policy Bindings <minio-operator-sts-policy-binding>` that matches the application.
+5. The PolicyBinding CRD returns the policy or policies that match the request, if any.
+6. The MinIO Operator sends the combined policy information for the application to the MinIO Tenant.
+7. The tenant creates temporary credentials matching the policy or policies for the request and returns those to the MinIO Operator.
+8. The MinIO Operator forwards the temporary credentials back to the application.
+9. The application uses the credentials to send the object storage calls to the MinIO tenant.
+
+Requirements
+------------
+
+STS for the MinIO Operator requires the following:
+
+- MinIO Operator v5.0.0 or later.
+- The deployment **must** have :ref:`TLS configured <minio-tls>`.
+- :envvar:`OPERATOR_STS_ENABLED` environment variable set to ``on``.
+
+Procedure
+---------
+
+1. Enable STS functionality for the deployment
+   
+   .. code-block:: shell
+      :class: copyable
+
+      kubectl -n minio-operator set env deployment/minio-operator OPERATOR_STS_ENABLED=on
+   
+   - Replace ``minio-operator`` with the namespace for your deployment.
+   - Replace ``deployment/minio-operator`` with the value for your deployment's MinIO Operator.
+
+     You can find the deployment value by running ``kubectl get deployments -n <namespace>``, where you replace ``<namespace>`` with the namespace for the MinIO Operator.
+     Your MinIO Operator namespace is typically ``minio-operator``, though this value can change during install.
+
+2. Ensure an appropriate :ref:`policy <minio-policy>` or policies exist on the MinIO Tenant for the application to use for the application
+
+   The next step uses a YAML document to map one or more existing tenant policies to a service account through a custom resource called a ``PolicyBinding``.
+
+3. Create YAML resources for the Service Account and Policy Binding: 
+
+   - The :ref:`Service Account <minio-operator-sts-service-account>` in the MinIO Tenant for the application to use.
+
+     For more on service accounts in Kubernetes, see the :kube-docs:`Kubernetes documentation <reference/access-authn-authz/service-accounts-admin/>`.
+   - Create a :ref:`Policy Binding <minio-operator-sts-policy-binding>` in the target tenant's namespace that link the application to one or more of the MinIO Tenant's policies.
+
+4. Apply the YAML file to create the resources on the deployment
+   
+   .. code-block:: shell
+      :class: copyable
+
+      kubectl apply -k path/to/yaml/file.yaml
+
+5. Use an SDK that supports the ``AssumeRoleWithWebIdentity`` like behavior to send a call from your application to the deployment
+
+   The STS API expects a JWT for the service account to exist in the Kubernetes environment.
+   This is typically mounted from a well known location on pods in the Kubernetes cluster, such as ``/var/run/secrets/kubernetes.io/serviceaccount/token``.
```

Comment:
For pods which specify the service account using <that field>, Kubernetes typically mounts that JWT at a well known location such as <thing>